bstats-velocity/src/examples/java/ExamplePluginInitEventHandler.java

```diff
@@ -0,0 +1,26 @@
+import java.util.concurrent.Callable;
+
+import com.velocitypowered.api.event.EventHandler;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+
+import org.bstats.velocity.Metrics;
+
+public class ExamplePluginInitEventHandler implements EventHandler<ProxyInitializeEvent> {
+    public void execute(ProxyInitializeEvent event) {
+        ExamplePlugin plugin = ExamplePlugin.getInstance();
+        
+        // All you have to do is adding this line in your onEnable method:
+        Metrics metrics = new Metrics(plugin.getProxy().getPluginManager().fromInstance(plugin).get(), plugin.getProxy(), plugin.getLogger());
```

Comment:
Calling `get()` on an Optional without checking if it's present is usually a bad idea.